loaf-explainer.md

```diff
@@ -223,6 +223,13 @@ const someLongAnimationFrameEntry = {
     // [timestamp](https://dom.spec.whatwg.org/#dom-event-timestamp), i.e. the time it was queued
     // which could be long before it was processed.
     firstUIEventTimestamp,
+    
+    // The duration in milliseconds that the LoAF was blocking in practice. Since a LoAF can be consisted
```

Comment:
Maybe something like:
"""
The duration in milliseconds that the animation frame was being blocked in practice.
Given that LoAFs can contain multiple tasks, we only consider as blocking:
* Long tasks.
* The longest task + the rendering time, if their sum exceeds the Long Task threshold of 50ms.
"""